charts/stable/zalando-postgres-cluster/templates/backupCronjob.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if .Values.dumpBackup.existingClaim -}}
+{{- if or .Values.dumpBackup.enabled .Values.dumpBackup.existingClaim -}}
```

Comment:
Did you put the `or` in here to make it a non-breaking change?

Reply:
exactly, it is to make it non-breaking